OrangeHRMTests/PageObjects/Elements/Tables.cs

```diff
@@ -0,0 +1,26 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Elements
+{
+    public static class Tables
```

Comment:
kind of Buttons class
if you need to implement class that will represent 'basic' app table, you need:
1. make it NON static
2. create instance of table on page level!!

example, you have a table on MainPage
you create public Table Table () => new Table() in PO of MainPage. then when you need to interact with this table you call it like mainPage.Table.ClickTrashButton() where mainPage - instance of MainPage

Reply:
Fixed